src/core/shape/attributes.js

```diff
@@ -10,50 +10,77 @@ import p5 from '../main';
 import * as constants from '../constants';
 
 /**
- * Modifies the location from which ellipses, circles, and arcs are drawn. By default, the
- * first two parameters are the x- and y-coordinates of the shape's center. The next
- * parameters are its width and height. This is equivalent to calling `ellipseMode(CENTER)`.
+ * Changes the way ellipses, circles, and arcs are drawn.
```

Comment:
How about 'Change where ellipses, circles, and arcs are drawn'? @nickmcintyre

Reply:
Updated!